packages/state-transition/test/unit/util/misc.test.ts

```diff
@@ -13,14 +13,14 @@ describe("getBlockRoot", () => {
     });
     const res = Buffer.from(getBlockRoot(state, GENESIS_SLOT));
     const expectedRes = BigInt("0xab");
-    assert(toBigIntLE(res) === expectedRes, `got: ${toBigIntLE(res)}, expected: ${expectedRes.toString(16)}`);
+    expect(toBigIntLE(res)).toEqual(expectedRes);
   });
   it("should fail if slot is current slot", () => {
     const state = generateState({slot: GENESIS_SLOT});
-    assert.throws(() => getBlockRoot(state, GENESIS_SLOT), "");
+    expect(() => getBlockRoot(state, GENESIS_SLOT)).toThrow("");
```

Comment:
Why does this function throw an empty string (`toThrow("")`)? Might be worth investigating

Reply:
Yes would be nice to investigate. But the fix/patch should be in separate PR.